src/Components/Patient/DailyRoundListDetails.tsx

```diff
@@ -11,23 +11,29 @@ const Loading = lazy(() => import("../Common/Loading"));
 const symptomChoices = [...SYMPTOM_CHOICES];
 const currentHealthChoices = [...CURRENT_HEALTH_CHANGE];
 
+function useConsultationQuery(consultationId: any, id: any) {
```

Comment:
Proper types are encouraged when you for sure know the types!

```suggestion
function useConsultationQuery(consultationId: string, id: string) {
```